x/cdp/types/genesis.go

```diff
@@ -17,10 +17,12 @@ type GenesisState struct {
 	DebtDenom                string    `json:"debt_denom" yaml:"debt_denom"`
 	GovDenom                 string    `json:"gov_denom" yaml:"gov_denom"`
 	PreviousDistributionTime time.Time `json:"previous_distribution_time" yaml:"previous_distribution_time"`
+	SavingsRateDistributed   sdk.Int   `json:"savings_rate_distributed" yaml:"savings_rate_distributed"`
```

Comment:
is this a param? should we remove the var definitions from that file? makes things more confusing

Reply:
It's not a param. I agree that having the definitions here is confusing and should probably be updated. But as this PR is scoped to the querier I just went with the existing module layout on this one.